migrate_pim/upgrade_from_32_to_40.rst

```diff
@@ -0,0 +1,156 @@
+Upgrade from 3.2 to 4.0
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Use this documentation to migrate projects based on the Enterprise Edition.
+
+Disclaimer
+**********
+
+Requirements
+************
+
+Akeneo PIM 4.0 Enterprise Archive
+---------------------------------
+
+TODO: How to download from the Portal
+
+
+Updated System components
+-------------------------
+
+   TODO: System stack migrated.
+
+
+Upgraded Virtual Host configuration
+-----------------------------------
+
+    TODO: link to new configuration
+
+
+
+Prepare your project
+********************
+
+In the following code, "my_pim" is the directory where the Akeneo PIM 3.2 you want to migrate is installed.
+
+.. code:: bash
+
+    $ cd my_pim
```

Comment:
In the previous upgrades we used to store the pim path in a variable:
`export PIM_DIR=/path/to/your/current/pim/installation`

in https://github.com/akeneo/pim-enterprise-dev/blob/3.2/UPGRADE.md

Reply:
I think it makes the reading of the documentation more cumbersome.

Reply:
you can copy/paste with a variable
which I guess this is what people expect (this is what I expect from a doc personally)

Reply:
I wanted to have a migration procedure easy enough so copy-pasted would not be necessary.
But I won't die on that hill ;)